package.json

```diff
@@ -50,30 +51,24 @@
         "eslint": "^8.2.0",
         "eslint-config-prettier": "^8.3.0",
         "eslint-plugin-prettier": "^4.0.0",
-        "husky": "7.0.4",
+        "husky": "8.0.3",
         "jasmine-core": "~3.10.1",
         "jasmine-spec-reporter": "~7.0.0",
         "karma": "~6.3.9",
         "karma-chrome-launcher": "~3.1.0",
         "karma-coverage-istanbul-reporter": "~3.0.3",
         "karma-jasmine": "~4.0.1",
         "karma-jasmine-html-reporter": "^1.7.0",
-        "lint-staged": "12.0.2",
+        "lint-staged": "13.1.1",
         "ng-packagr": "^13.0.3",
         "prettier": "2.4.1",
         "protractor": "~7.0.0",
         "ts-node": "~10.4.0",
         "typescript": "~4.4.3"
     },
-    "husky": {
-        "hooks": {
-            "pre-commit": "lint-staged"
-        }
-    },
     "lint-staged": {
         "*.{json,md,ts,html,component.html}": [
-            "prettier --write",
-            "git add"
```

Comment:
Manual add not needed anymore since v10:

https://github.com/okonet/lint-staged/releases/tag/v10.0.0

> Prior to version 10, tasks had to manually include git add as the final step. This behavior has been integrated into lint-staged itself in order to prevent race conditions with multiple tasks editing the same files. If lint-staged detects git add in task configurations, it will show a warning in the console. Please remove git add from your configuration after upgrading.